src/components/Table/Table.jsx

```diff
@@ -131,6 +133,7 @@ class Table extends React.Component {
       placeholderMessage,
       pageLength = 20,
       hoverRowColumn,
+      highlightFn = function () {},
```

Comment:
I don't think you should construct a new function by default, just let it be null and check for null before you try to run the function